UX-Guide-Metadata/principles/index.html

```diff
@@ -96,7 +96,7 @@
 		<section id="general-overview">
 			<h2>General Overview</h2>
 			<p>This document will help those who wish to provide accessibility metadata directly to users understand how
-				to represent machine readable accessibility metadata in a user friendly User Interface / User Experience
+				to represent machine readable accessibility metadata in a user-friendly User Interface / User Experience
```

Comment:
machine-readable should be hyphenated, too.

Reply:
Ok just pushed that change :)